community/modules/scheduler/htcondor-configure/README.md

```diff
@@ -72,9 +72,81 @@ example can be found in the [examples README][htc-example].
   - internal_ip
 ```
 
-A full example can be found in the [examples README][htc-example].
```

Comment:
FYI: this text appears above the YAML snippet now. So this is removing duplicated text.